content/xkcd.md

```diff
@@ -9,6 +9,6 @@ or
 <%= @item[:description] %>
 ====================
 
-![Obligatory xkcd cartoon](http://imgs.xkcd.com/comics/e_to_the_pi_minus_pi.png "Obligatory xkcd cartoon")
+![强制的 xkcd 漫画](http://imgs.xkcd.com/comics/e_to_the_pi_minus_pi.png "Obligatory xkcd cartoon")
```

Comment:
*Obligatory*可能并不是*强制*的意思，请参考：[Dict](http://dictionary.cambridge.org/zhs/%E8%AF%8D%E5%85%B8/%E8%8B%B1%E8%AF%AD/obligatory) C2: "expected because it usually happens"

建议翻成："**按照惯例，我们请出 xkcd 漫画**"